src/config.rs

```diff
@@ -1542,33 +1565,78 @@ impl DBConfigManger {
                  block-cache.capacity in storage module instead"
                 .into());
         }
-        let opt = self.db.get_options_cf(cf)?;
-        opt.set_block_cache_capacity(size.0)?;
+        let mut result = Ok(());
+        self.tablet_factory
+            .loop_tablet_cache(Box::new(|region_id, suffix, db: &RocksEngine| {
+                let r = db.get_options_cf(cf);
+                if let Ok(opt) = r {
+                    let r = opt.set_block_cache_capacity(size.0);
+                    if r.is_err() {
+                        result = Err(Box::from(r.err().unwrap()));
+                        error!(
+                            "set_block_cache_size {} failed on tablet {} {}. err:{:?}",
+                            size.as_mb(),
+                            region_id,
+                            suffix,
+                            &result
+                        );
+                    }
+                } else {
+                    result = Err(Box::from(r.err().unwrap()));
```

Comment:
Do we need to print error log for failure get_options_cf() ?

Reply:
good catch. I think so.